connector/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/JsonUtils.scala

```diff
@@ -65,9 +68,9 @@ private object JsonUtils {
   def partitionOffsets(str: String): Map[TopicPartition, Long] = {
     try {
       Serialization.read[Map[String, Map[Int, Long]]](str).flatMap { case (topic, partOffsets) =>
-          partOffsets.map { case (part, offset) =>
-              new TopicPartition(topic, part) -> offset
-          }
+        partOffsets.map { case (part, offset) =>
```

Comment:
Please do not touch unrelated code.

Reply:
I used dev/scalafmt because there were quite a few modified files, but it seems that this tool is not suitable for old code.I will roll back

Reply:
Currently, only the connect-related modules are required to use scalafmt.